.env.example

```diff
@@ -38,14 +39,29 @@ PUSHER_APP_CLUSTER=mt1
 MIX_PUSHER_APP_KEY="${PUSHER_APP_KEY}"
 MIX_PUSHER_APP_CLUSTER="${PUSHER_APP_CLUSTER}"
 
-GITHUB_CLIENT_ID=
-GITHUB_CLIENT_SECRET=
-GITHUB_REDIRECT_URL=
 
-GOOGLE_CLIENT_ID=
-GOOGLE_CLIENT_SECRET=
-GOOGLE_REDIRECT_URL=
+# Set these up at https://github.com/settings/applications/new
```

Comment:
Like this a lot! Good idea to add these urls to the env

Reply:
Took me a while to get this working, so I'm glad you like it @fritsvt